src/native/libs/System.IO.Compression.Native/zlib_allocator.h

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifdef _WIN32
+    #include <stdint.h>
+    #include <windows.h>
+    #define FUNCTIONEXPORT
+    #define FUNCTIONCALLINGCONVENTION WINAPI
+#else
+    #include "pal_types.h"
+    #include "pal_compiler.h"
+    #define FUNCTIONEXPORT PALEXPORT
+    #define FUNCTIONCALLINGCONVENTION
+#endif
+#include <zconf.h> // voidpf
+
+FUNCTIONEXPORT voidpf FUNCTIONCALLINGCONVENTION z_custom_calloc(voidpf opaque, unsigned items, unsigned size);
+
+FUNCTIONEXPORT void FUNCTIONCALLINGCONVENTION z_custom_cfree(voidpf opaque, voidpf ptr);
```

Comment:
```suggestion
voidpf FUNCTIONCALLINGCONVENTION z_custom_calloc(voidpf opaque, unsigned items, unsigned size);

void FUNCTIONCALLINGCONVENTION z_custom_cfree(voidpf opaque, voidpf ptr);
```
This should not be `FUNCTIONEXPORT`

Reply:
I can remove them and I should also remove lines 7 and 12.

I was imitating pal_zlib.h, which has the FUNCTIONEXPORT prefix in the methods.

Reply:
These are internal helper methods. They are not meant to be used from outside. They are fundamentally different from pal_zlib.h methods are meant to be used from outside by the managed code.

(I am not sure why you need FUNCTIONCALLINGCONVENTION either. That can be probably deleted too.)